test/spec/model/ModelSpec.js

```diff
@@ -1,8 +1,13 @@
 import {
   create,
-  isModelElement
+  isElementInstance
```

Comment:
Why would we rename this?

Reply:
We don't have to. One thing I was wondering about is the fact we use this check in `Modeling`. Doesn't that have the same issue with having more than one diagram-js?

Reply:
Yes, it does. However it was never a problem to date. A safe way is to rather do the check in the `elementFactory`, hence my prior attempt to move it there.

Reply:
Reverted.